doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -8,226 +8,72 @@ A step by step guide to the GeoServer Compliance Interoperability Test Engine (C
-   ``WEB-INF/lib`` directory.
+   .. note::
+      Valid values for the ETS parameter are
+        * wcs10
+        * wcs11
+        * wfs10
+        * wfs11
+        * wms11
+        * wms13
    
-#. Remove the ``wcs1_1-<VERSION>.jar`` from ``WEB-INF/lib`` directory.
-
-#. Start GeoServer with the ``citewcs-1.0`` data directory.
-
-#. Change directory back to the cite tools and run the tests::
-
-     ant wcs-1.0
-     
-   With the following parameters:
-
-   #. ``Capabilities URL``:
-        
-          http://localhost:8080/geoserver/wcs?service=wcs&request=getcapabilities&version=1.0.0
-
-   #. ``MIME Header Setup``: "image/tiff"
-
-   #. ``Update Sequence Values``:
-
-      * "2" for ``value that is lexically higher``
-      * "0" for ``value that is lexically lower``
-
-   #. ``Grid Resolutions``:
+   Choose which GeoServer war to test by setting the GEOSERVER_WEBAPP_SRC environment variable::
 
-      * "0.1" for ``RESX``
-      * "0.1" for ``RESY``
+    GEOSERVER_WEBAPP_SRC=https://build.geoserver.org/geoserver/master/geoserver-master-latest-war.zip
 
-   #. ``Options``:
+#. Build the testing suite::
   
-      * Ensure ``Verify that the server supports XML encoding`` is *checked*
-      * Ensure ``Verify that the server supports range set axis`` is *checked*
-
-   #. ``Schemas``:
-
-      * Ensure that ``original schemas`` is selected
-
-   #. Click ``OK``
-
-   .. image:: tewcs-1_0a.jpg
-
-   .. image:: tewcs-1_0b.jpg
-
-.. _commandline:
+    # FROM THE GEOSERVER CODE BASE ROOT FOLDER
+    pushd build/cite
+    docker_command="docker-compose -f docker-compose.yml -f ./${ETS}/docker-compose.override.yml "
+    ### Cleaning up previous runs
+    eval $docker_command  down --rmi all -v
+    eval $docker_command  rm -vfs
+    
+    ### Build docker images
+    eval $docker_command  build --build-arg "GEOSERVER_WEBAPP_SRC=${GEOSERVER_WEBAPP_SRC}" geoserver
+
+#. Run the test::
+  
+    eval $docker_command  up --force-recreate teamengine
 
-Teamengine Web Application
---------------------------
+How to run TEAM Engine standalone
+---------------------------------
+To run a standalone version of TEAM Engine, start it with the following command::
 
-The Teamengine web application is useful for analyzing results of a test run. To run the web application execute::
+    docker run -d --name standalone_teamengine -p 8080:8080 geosolutionsit/teamengine:latest
 
-  ant webapp
-  
-From the cite tools checkout. Once started the web app will be available at:
+TEAM Engine will be accessible on http://localhost:8080/teamengine/
 
-  http://localhost:9090/teamengine
-  
-To run on a different port pass the ``-Dengine.port`` system property to ant command.
+If you want to change the port, for example to have it on port "9090", change the command as follows::
 
+    docker run -d --name standalone_teamengine -p 9090:8080 geosolutionsit/teamengine:latest
 
+To stop TEAM Engine::
 
+    docker stop standalone_teamengine
```

Comment:
The deleted line indicates this was useful to review the test results? Is that still the case? How to we review the test results ....